src/Tracker/index.js

```diff
@@ -47,18 +27,38 @@ const setUp = () => {
   // This is so we can track the original page where the user opened the SDK.
   woopra.identify(client.match(/^(id|id-dev)\.onfido\.com$/) ?
     {sdk_version} : {sdk_version, client})
-
-  Raven.TraceKit.collectWindowErrors = true//TODO scope exceptions to sdk code only
 }
 
 const uninstall = () => {
-  RavenTracker.uninstall()
+  const sentryClient = Sentry.getCurrentHub().getClient();
+  if (sentryClient) {
+    sentryClient.close(2000)
+  }
   woopra.dispose()
   shouldSendEvents = false
 }
 
 const install = () => {
-  RavenTracker.install()
+  Sentry.init({
```

Comment:
Sorry, i was just thinking, can this init of sentry mess up clients that also use sentry to track their own web apps?

Reply:
@stefaniacardenas any idea about this? Did it already did that?

Reply:
I think you've got a point there. I will make sure our Sentry integration does not clash with our client's.

Reply:
out of curiosity how would we ensure this? 🤔

Reply:
I created multiple instances of Sentry and I noticed that the second instance was overriding the first one. Luckily, Sentry has a way of fixing this by using `BrowserClient`.